media/css/firefox/features/index.scss

```diff
@@ -2,386 +2,44 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../pebbles/includes/lib';
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
 
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/templates/card-layout';
 
-.main-header {
-    position: relative;
-
-    p {
-        margin: 20px auto 40px;
-    }
-
-    &:after {
-        background: transparent url('/media/img/firefox/features/quantum/wave.png') center center no-repeat;
-        @include background-size(5223px 186px);
-        bottom: -30px;
-        content: '';
-        display: block;
-        height: 186px;
-        left: 0;
-        position: absolute;
-        width: 100%;
-    }
-
-    .content {
-        padding-bottom: 60px;
-    }
-
-    @media #{$mq-tablet} {
-        padding-bottom: 0;
-    }
-}
-
-.header-image {
-    display: none;
-    margin-bottom: 40px;
-
-    img {
-        max-width: 500px;
-        width: 100%;
-    }
-
-    @media #{$mq-tablet} {
-        display: block;
-        float: right;
-        margin-left: 10%;
-        width: 60%;
-
-        img {
-            max-width: 721px;
-        }
-    }
-}
-
-.header-content {
-    h1 {
-        @include at2x('/media/protocol/img/logos/firefox/browser/logo-lg.png', 100px, 100px);
-        background-position: top center;
-        background-repeat: no-repeat;
-        padding-top: 120px;
-    }
-
-    @media #{$mq-tablet} {
-        float: left;
-        text-align: left;
-        width: 30%;
-
-        h1 {
-            background-position: top left;
-        }
-    }
+.mzp-c-hero.mzp-has-image {
+    min-height: 450px;
 }
 
-@media #{$mq-tablet} {
-    html[dir="rtl"] {
-        .header-image {
-            float: left;
-            margin-left: 0;
-            margin-right: 10%;
-        }
+.section-title {
+    text-align: center;
 
-        .header-content {
-            float: right;
-            text-align: right;
-        }
+    @media #{$mq-md} {
+        margin: $spacing-xl 0;
     }
 }
 
-.fx-product-features-section {
-    background: #f9f9fa;
-    padding-top: 3em;
-
-    h2 {
-        @include font-size-level3;
-        color: $color-quantum-blue;
-        font-weight: bold;
-        margin-bottom: 1em;
-        text-align: center;
-
-        &:before {
-            background: $color-quantum-blue;
-            border-radius: 5px;
-            content: '';
-            height: 5px;
-            left: 50%;
-            margin-left: -1.2em;
-            position: absolute;
-            top: 0;
-            width: 2.4em;
-        }
-    }
+section.features.mzp-l-card-third {
```

Comment:
We should try to avoid qualifying selectors with elements, just the class(es) should be enough without tying it to a particular element.